bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/config/CallOptionsConfig.java

```diff
@@ -104,6 +112,19 @@ public Builder setShortRpcTimeoutMs(int shortRpcTimeoutMs) {
       return this;
     }
 
+    /**
+     * The amount of milliseconds to wait before issuing a client side timeout for short RPCs.
```

Comment:
Please expand this comment to state that how this differs from shortRpcTimeout. ie something along the lines of "Unlike shortRpcTimeout, this limits how long each attempt will take, whereas shortRpcTimeout will limit the total amount across all of the attempts".

Also please mention what short means here. ie "This setting affects point reads and writes like checkAndMutate, mutateRow and readRows with single keys"

Reply:
Done, and updated the other new methods too